hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2730,7 +2730,7 @@ def hpat_pandas_series_argsort_noidx_impl(self, axis=0, kind='quicksort', order=
         na = self.isna().sum()
         result = numpy.empty(len(self._data), dtype=numpy.int64)
         na_data_arr = hpat.hiframes.api.get_nan_mask(self._data)
-        sort_nona = numpy.argsort(self._data[~na_data_arr])
+        sort_nona = numpy.argsort(self._data[~na_data_arr], kind='mergesort')
```

Comment:
I think it's not correctly to use `'mergesort'` when input `kind='quicksort'`.

Reply:
We cannot use 'quicksort' sorting because then we will not compare indices

Reply:
Let's imagine we already support parameter `kind`. So are you still going to sort data via `'mergesort'`?

Reply:
When other sortings are implemented, it will be possible to change this parameter, but now it’s more efficient to leave it and not spend much time on it

Reply:
Maybe I should write that now only mergesort is supported

Reply:
Anyway I vote for unification of parameter `kind` for `pd.argsort` and `np.argsort` to be able to expand the functionality simpler in further, in particular to add support of parameter `kind`.